content/docs/user-guide/gems/reference/audio/wwise/audio-engine-wwise.md

```diff
@@ -25,6 +25,12 @@ In general you can use more recent versions of Wwise than specified above, but s
 
 1. (Recommended) Set the CMake cache variable `LY_WWISE_INSTALL_PATH` to the path where you installed Wwise. You can use `cmake-gui` to set this variable, or you can set it when you run the `cmake` build configuration command. By using this cache variable, CMake project regeneration will automatically be triggered if the variable is updated in the future.
 
+    You can use the following CMake build configuration command to set `LY_WWISE_INSTALL_PATH` from your source engine or project directory.

```

Comment:
I think we should focus on the project configuration. I don't see a common use case for running it from the source engine directory.
```suggestion
    You can use the following CMake build configuration command to set `LY_WWISE_INSTALL_PATH` from your project directory.

```